psi4/src/psi4/libmints/wavefunction.h

```diff
@@ -403,6 +412,20 @@ class PSI_API Wavefunction : public std::enable_shared_from_this<Wavefunction>
     /// Returns the core Hamiltonian matrix
     SharedMatrix H() const { return H_; }
 
+    /// Sets the alpha electrons MO coefficients
+    void set_Ca(SharedMatrix& Ca_new);
```

Comment:
I think we have explicitly avoided `set_Ca` to prevent confusion and some possible errors. Is there a reason that `wfn->Ca()->clone(new_Ca)` will not work for you as a way of override?

Reply:
How would that work from the python side? This does not seem to work for me:
    wfn = psi4.core.legacy_wavefunction()
    new_ca = wfn.Ca()
Neither `new_ca` nor `wfn.Ca()` has a clone function which accepts another matrix to clone from. This was the reason for adding the setter functions, which are then exposed to python as well, so that it can be set from there. Is there another way to do this?

Reply:
Sorry `->copy(new_Ca)`.